typescript/infra/scripts/funding/fund-keys-from-deployer.ts

```diff
@@ -116,6 +119,46 @@ const desiredBalancePerChain: ChainMap<string> = {
   test3: '0',
 };
 
+// Used to fund kathy with more tokens such that it's able to pay interchain gas
+// on mainnet. The amount is roughly > $100
+const desiredKathyBalancePerChain: ChainMap<string> = {
```

Comment:
Can we use ChainMap<MainnetChains> or st similar here?

Reply:
I think following the type loosening in https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/1804/files#diff-6900eeda572fe4397d5305d20de146fafd4009dc257b16ccc81f17b9c17c896cR6 this is the best we can do